Misc/NEWS.d/next/Library/2020-04-27-14-48-43.bpo-39966.N5yXUe.rst

```diff
@@ -0,0 +1,2 @@
+Revert bpo-25597. :class:`unittest.mock.MagicMock` with wraps' set uses
+default return values for magic methods.
```

Comment:
Not sure we need this if the new item is removed above?

Reply:
The change was published as part of an alpha release. So my belief with respect to reversion after a release is that someone might depend on it and to explicitly note tbe reversion as a NEWS entry from past reversions I looked in git log.

Reply:
Agreed, but perhaps that means the original news item should stay too? It's a little confusing to have a reference to a bpo that isn't anywhere else in the changelog.

Reply:
Makes sense, added it back. Each alpha has a separate section and it reads like added in alpha 4 to be reverted by alpha 6. If anyone has concern or feedback later I will change it.